package.json

```diff
@@ -29,9 +29,10 @@
     "./tests/test.js": "./tests/browser/test.js"
   },
   "dependencies": {
-    "blakejs": "^1.1.1",
-    "bs58check": "^2.1.2",
+    "blakejs": "github:aeternity/blakejs",
```

Comment:
I'd like to see a fixed release version here, also a semver compliance of this lib

Reply:
on tuesday the 15.03. we will either take the updated version of the lib or release and publish our own fork on npm, see https://github.com/dcposch/blakejs/pull/19#issuecomment-1065015716

Reply:
@dincho just in case the contributor there could create a specific tag for the library on github - would it be ok to rely on that or do we want to have it published on NPM? see discussion here: https://github.com/dcposch/blakejs/issues/20

Reply:
NPM releases have one very strong property: https://docs.npmjs.com/policies/unpublish
So yes, I insist on NPM release dependancy.